src/contour/Audio.h

```diff
@@ -14,6 +14,13 @@
 #endif
 namespace contour
 {
+
+#if QT_VERSION >= 0x060000
```

Comment:
I think we can use QT_VERSION_CHECK macro here, I think we use it already in some places

Reply:
true. i simply moved that line and didn't think about if this line actually could be written better, will do, thanks.